src/main/java/juja/microservices/users/dao/crm/domain/UserCRM.java

```diff
@@ -63,4 +69,6 @@ public UserCRM() {
 | c_slack            | varchar(255)     | YES  |     | NULL    |                |
 | c_isStudent        | tinyint(1)       | NO   |     | 0       |                |
 | c_uuid             | varchar(255)     | YES  |     | NULL    |                |
+| assignedTo         | varchar(50)      |      |     |         |                |
+| visibility         | int(11)          |      |     |         |                |
```

Comment:
what about null and default attributes?

Reply:
@danilkuznetsov Your turn!

Reply:
@VadimDyachenko 

| visibility         | int(11)          | NO   |     | NULL    |                |
| assignedTo         | varchar(50)      | YES  | MUL | NULL    |                |

Reply:
Done